pkg/minikube/command/ssh_runner.go

```diff
@@ -182,13 +196,58 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	})
 
 	scp := fmt.Sprintf("sudo mkdir -p %s && sudo scp -t %s", f.GetTargetDir(), f.GetTargetDir())
+	mtime, err := f.GetModTime()
+	if err != nil {
+		glog.Infof("error getting modtime for %s: %v", f.GetAssetName(), err)
```

Comment:
Use `dst` instead of `f.GetAssetName()`, as the latter may be empty :(

Also, skip if `mtime.IsZero()`

Reply:
done!